### ISSUE TITLE
Test run finished: 0 Tests (0 Passed, 0 Failed, 0 Skipped) run in < 1 ms

### ISSUE THREAD

**Issue by Legends** [microsoft/testfx#1599]
## Describe the bug

.net 7 class library (x64 / Debug)
.net 7 test project (x64 / Debug)

TestClass with one test method.

Right.click test method => Debug

![image](https://user-images.githubusercontent.com/966368/220109639-13e08de2-e508-42f5-8431-e2518b817d90.png)

Building Test Projects
========== Starting test run ==========
========== Test run finished: 0 Tests (0 Passed, 0 Failed, 0 Skipped) run in < 1 ms ==========

## Additional context
- Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.4.5

![image](https://user-images.githubusercontent.com/966368/220110049-2aeccb84-a656-4be4-b8da-70af19b8b40d.png)

**Comment by Legends** [comment]
Ok, really weird behaviour.

I have a: 
```
[ClassInitialize]
public static void Initialize(){...}
```

I can remember that I had this problem a couple of years ago and I assumed that the Initialize method signature is wrong.
At that time It didn't tell me as well the reason for not executing tests.
So I commented it out. The I got another exception message, can't remember the error message.
I comment in the Initialize method and correct the signature.
Build test project and debug test method works now.


It should really tell, whats the issue and not just not finding tests.
